packages/storybook/examples/core/IconButton.stories.js

```diff
@@ -3,7 +3,18 @@ import React from 'react';
 import IconButton from '@ichef/gypcrete/src/IconButton';
 import FlexRow from 'utils/FlexRow';
 
-function IconButtonWithStatusExample() {
+export default {
+    title: '@ichef/gypcrete|IconButton',
+    component: IconButton,
```

Comment:
Since it's a variant of `<Button>`, please also help include props table of `<Button>` here (via `subcomponents`)